composer.json

```diff
@@ -18,7 +19,7 @@
 		"GPL-2.0-or-later"
 	],
 	"require": {
-		"typo3/cms-core": "^9 || ^10 || ^11"
+		"typo3/cms-core": "^9 || ^10 || ^11 || ^12"
```

Comment:
should we skip support for 9 & 10?

Reply:
i would recommend it, but i don't want to take that decision on my one